### PR TITLE
docs: fix issues with templates

### PR DESCRIPTION
**Pull request by jayu**
## Motivation

My previous PR introduced some formating issue for bug issue template

## Summary

issue template settings are YAML file, I used some validator and fixed it :D

## Test plan

I copy-pasted new versions in file editor on Github,  it recognizes bug template correctly

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ---
-name: Report a bug 🐛
-about: Report a reproducible bug or reproducible regression.
-labels: 'bug report 🦗', 'needs: triage 🏷', 'needs: complete repro 🖥️'
+name: "Report a bug 🐛"
+about: "Report a reproducible bug or reproducible regression."
+labels: "bug report 🦗, needs: triage 🏷, needs: complete repro 🖥️"
```
Comment (thymikee) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: You don't need quotation marks at all
```suggestion
name: Report a bug 🐛
about: Report a reproducible bug or reproducible regression.
labels: bug report 🦗, needs: triage 🏷, needs: complete repro 🖥️
```

Comment (jayu) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: labels don't work without quotation marks and it cannot be parsed correctly

Comment (thymikee) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: Weird, it works on Jest repo

Comment (jayu) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: This is becoause of emojis

Comment (jayu) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: @thymikee can we proceed? Currently, there is no issue template for bug😅

**Pull request merged**